test/ios_framework_import_test.sh

```diff
@@ -202,7 +202,16 @@ let sharedClass = SharedClass()
 sharedClass.doSomethingShared()
 EOF
 
-    do_build ios //app:app || fail "Should build"
+    do_build ios --compilation_mode=dbg //app:app || fail "Should build"
+
+    unzip_single_file "test-bin/app/app.ipa" "Payload/app.app/app" |
+        nm -aj - | \
+            grep -e /app_main.swiftmodule \
+                 -e /iOSSwiftStaticFramework.swiftmodule/x86_64.swiftmodule | \
+            wc -l | \
+            grep -e '^\s*2\s*$' > /dev/null \
+            || fail "Could not find .swiftmodule AST references in binary; " \
+                    "linkopts may have not propagated"
```

Comment:
I tacked this test on to an existing test. Hopefully this is ok, the main thing I'm unsure of is that I forced compilation mode to `dbg`. How do these tests get a compilation mode? Do they get called with each compilation mode?

Reply:
Also, suggestions welcome for how to structure this shell code 😬